Makefile

```diff
@@ -47,49 +30,60 @@ endif
 #     etc/
 #     share/kafka/
 ifndef PREFIX
-PREFIX=$(DEFAULT_PREFIX)
+PREFIX=$(PACKAGE_NAME)
+endif
+
+# Whether we should run tests during the build.
+ifndef SKIP_TESTS
+SKIP_TESTS=no
 endif
 
 ifndef SYSCONFDIR
-SYSCONFDIR:=$(DEFAULT_SYSCONFDIR)
+SYSCONFDIR=PREFIX/etc/$(PACKAGE_TITLE)
+endif
+
+# Whether we pull artifacts from nexus/artifactory
+ifndef PULL_ARTIFACTS
+PULL_ARTIFACTS=no
```

Comment:
You can set default value to shell variables as in like here:
https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash

if env variable is available use it else use default.

Reply:
Currently, I don't want to modify too much for `Makefile`, incase it breaks other stuff during packaging that I am not familiar with. However, I think we do need to simplify them later.